src/main/java/uk/gov/hmcts/reform/cwrdapi/controllers/CaseWorkerRefController.java

```diff
@@ -76,4 +78,52 @@
         }
         return caseWorkerService.processCaseWorkerProfiles(caseWorkersProfileCreationRequest);
     }
+
+    @ApiOperation(
+            value = "This API builds the idam role mappings for case worker roles",
+            authorizations = {
+                    @Authorization(value = "ServiceAuthorization"),
+                    @Authorization(value = "Authorization")
+            }
+    )
+    @ApiResponses({
+            @ApiResponse(
+                    code = 201,
+                    message = "Successfully built idam role mappings for case worker roles",
+                    response = IdamRolesMappingResponse.class
+            ),
+            @ApiResponse(
+                    code = 400,
+                    message = "Bad Request"
+            ),
+            @ApiResponse(
+                    code = 401,
+                    message = "Unauthorized Error : The requested resource is restricted and requires authentication"
+            ),
+            @ApiResponse(
+                    code = 403,
+                    message = "Forbidden Error: Access denied for invalid permissions"
+            ),
+            @ApiResponse(
+                    code = 500,
+                    message = "Internal Server Error"
```

Comment:
Would be good if we could change these common error response messages into constants for future APIs

Reply:
done